app/Http/Middleware/Admin.php

```diff
@@ -16,6 +19,44 @@ class Admin
      */
     public function handle($request, Closure $next)
     {
+        if (Auth::user()) {
+            $user = User::findOrFail(Auth::id())->load(['requests' => function ($query) {
+                $query->where('status', '<>', config('request.reject'))
+                    ->where('status', '<>', config('request.return'))
+                    ->where('status', '<>', config('request.late'))
+                    ->where('status', '<>', config('request.forget'));
+            }]);
+            $status = $user->status;
+            foreach ($user->requests as $item) {
```

Comment:
Em xem như thế này có được ko nhé
```suggestion
            $status = [config('request.reject'), config('request.return'), config('request.late'), config('request.forget')];
            $user = User::findOrFail(Auth::id())->load(['requests' => function ($query) {
                $query->whereNotIn('status', $status);
            }]);
            $status = $user->status;
            foreach ($user->requests as $item) {
```

Reply:
Được anh ạ! Thanks anh nha :D